README.md

```diff
@@ -24,6 +24,11 @@ paper](http://adsabs.harvard.edu/abs/2011ApJS..192....9T) you may be interested
 in; if you use yt in the preparation of a publication, please consider citing
 it.
 
+## Code of Conduct
+
+yt abides by a code of conduct partially modified from the PSF code of conduct,
+and is found [in our contributing guide](CONTRIBUTING.rst).
```

Comment:
Can you link to the doc page? That'll let you deeplink directly to the CoC:

http://yt-project.org/docs/dev/developing/developing.html#yt-community-code-of-conduct